mobile/ios/Podfile.lock

```diff
@@ -191,6 +191,8 @@ PODS:
     - OpenSSL-Universal/Static (= 1.0.2.20)
   - OpenSSL-Universal/Static (1.0.2.20)
   - PromisesObjC (1.2.12)
+  - RCTAppleHealthKit (0.8.0):
```

Comment:
@rlnsy Can you confirm this is meant to be part of this PR? I thought this PR is only for updating the readme?

Reply:
Removed